README.md

```diff
@@ -81,6 +82,12 @@ grafton test --product=bonnets --plan=small --region=aws::us-east-1 \
     http://localhost:4567
 ```
 
+### Excluding Features
```

Comment:
So SSO is handled this way too?
It's all opt-out rather than opt-in?

Reply:
Yeah, any test can be disabled. We can only exclude right now, but we could add an `include` flag too.